docs/datagrid/general.md

```diff
@@ -23,71 +23,3 @@ By incorporating this component, users can effectively interact with data in a s
     ```typescript
     import * as DataGrid from '@highcharts/dashboards/datagrid';
     ```
-
```

Comment:
We should leave the introduction, or add it in the future. What is the reason of removing it?

Reply:
I guess it's a part of the documentation task.